CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
-## 4.0.1
```

Comment:
This shouldn't be removed right?

Reply:
oops, not sure how that happened. good catch.

Reply:
fixed